### ISSUE TITLE
Tool Upgrades

### ISSUE THREAD

**Issue by elifoster** [Esteemed-Innovation/Flaxbeards-Steam-Power-Resources#2]
@Flaxbeard Can you explain your naming scheme for the upgrades? I don't really understand them.

**Comment by Flaxbeard** [comment]
The 0 or 1 is the frame, just like drill0 and drill1 currently. The name before is the drill slot upgrade (bfd is the 3x3 miner, the rest are pretty straightforward. Since we have no precedent for 2 upgrades on one item, I just put the second upgrade after the 1/0. Furnace is autosmelt, overclock is +speed, processor is the smasher. The combination of upgrades could potentially be rendered automatically but I provided already combined sprites just in case.

Also unrelated but I couldn't reach you on IRC this morning - I applied the rewrite of the git log on the three branches specified (features-steamtools, exo-cosmetics, and features-automation).
